Formula/wsk.rb

```diff
@@ -0,0 +1,27 @@
+class Wsk < Formula
+  desc "OpenWhisk Command-Line Interface (CLI)"
+  homepage "http://openwhisk.org"
+  url "https://github.com/apache/incubator-openwhisk-cli/archive/0.9.0-incubating.tar.gz"
+  version "0.9.0-incubating"
+  sha256 "e2101daae8c4fdd43bb94bc9f8a2a29b74ce0a320b72e162f7f519b4dcbdd9b0"
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build
+  depends_on "gradle" => :build
+
+  def install
+    system "sed", "-i.bak", "s/4\.1/4.9/", "./gradle/wrapper/gradle-wrapper.properties"
+    ENV["GOPATH"] = buildpath
+    (buildpath / "src/github.com/apache/incubator-openwhisk-cli").install buildpath.children
+    cd "src/github.com/apache/incubator-openwhisk-cli" do
+      system "go-bindata", "-pkg", "wski18n", "-o", "wski18n/i18n_resources.go", "wski18n/resources"
+      system "./gradlew", "goVendor"
+      system "go", "build", "-o", "wsk"
+      bin.install "wsk"
+    end
+  end
+
+  test do
+    system "#{bin}/wsk", "property", "set", "--apihost", "https://127.0.0.1"
```

Comment:
This test changes configuration! It wipes out one's existing apihost in `~/.wskprops` (if present).

I would suggest a read-only command like `wsk namespace list`.

Reply:
Homebrew captures `HOME` for the duration of `def install` and `test do` and redirects it to `buildpath`/`testpath`.

Reply:
Also, `wsk` should come with a `--version` option like all cli tools, that allows to see the actual version.

Reply:
Ok, thanks, two things I didn't know!

Reply:
No worries @alexkli! I had no idea up until a couple days ago either 😄

Reply:
agreed we should add --version upstream - `wsk property get --cliversion` is available atm.